pkgs/applications/version-management/scmpuff/default.nix

```diff
@@ -20,6 +20,11 @@ buildGoModule rec {
     command = "scmpuff version";
   };
 
+  propagatedUserEnvPkgs = [
```

Comment:
Would be better to use wrapProgram to add PATH.

Reply:
It should work, as the spawned process will inherit environment variables. You can also try substitution anyway. Propagating doesn't seem to be that popular in this area...